test/celeritas/ext/Vecgeom.test.cc

```diff
@@ -1460,6 +1472,33 @@ TEST_F(SolidsGeantTest, reflected_vol)
 
 //---------------------------------------------------------------------------//
 
+TEST_F(ZnenvGeantTest, trace)
+{
+    // NOTE: This tests the capability of the G4PVDivision conversion based on
+    // an ALICE component
+    static char const* const expected_mid_volumes[]
+        = {"World", "ZNENV", "ZNST", "ZNST",  "ZNST", "ZNST", "ZNST",
+           "ZNST",  "ZNST",  "ZNST", "ZNST",  "ZNST", "ZNST", "ZNST",
+           "ZNST",  "ZNST",  "ZNST", "ZNST",  "ZNST", "ZNST", "ZNST",
+           "ZNST",  "ZNST",  "ZNST", "ZNENV", "World"};
+    static real_type const expected_mid_distances[]
+        = {6.38, 0.1,  0.32, 0.32, 0.32, 0.32, 0.32, 0.32, 0.32,
+           0.32, 0.32, 0.32, 0.32, 0.32, 0.32, 0.32, 0.32, 0.32,
+           0.32, 0.32, 0.32, 0.32, 0.32, 0.32, 0.1,  46.38};
```

Comment:
These results are from the geant4 navigator; the geometry is a subset of the ZDC input that has the division volumes.